packages/wallets/src/components/Base/WalletAlertMessage/WalletAlertMessage.scss

```diff
@@ -27,10 +27,12 @@
         }
     }
 
-    &__message-container {
-        display: flex;
-        align-items: center;
-        width: 100%;
-        padding: 0.8rem 0;
+    &__message {
+        &-container {
+            display: flex;
+            align-items: center;
+            width: 100%;
+            padding: 0.8rem 0;
+        }
```

Comment:
This was fine as it is before. I don't see any benefits of nesting it inside of `__message`

Reply:
Updated uwu